### ISSUE TITLE
NPM version outdated

### ISSUE THREAD

**Issue by furliq** [azain/electron-window-manager#82]
Is there a reason this package hasn't been published to npm? I'd like to use this package but I have to clone it because I need the latest version.

Thanks.

**Comment by a-sync** [comment]
There are no tests or active maintainers to make sure features work as intended. (i don't want to break anyone's code by publishing an untested version)   

You can add the github repo directly as a dependency in npm.  
eg.: `npm i tamkeen-tms/electron-window-manager#master`

**Comment by furliq** [comment]
@a-sync Ok thanks for the info. See this mentioned in PR https://github.com/tamkeen-tms/electron-window-manager/pull/81 too.

Is there any preferred approach to writing tests?

**Comment by a-sync** [comment]
Nothing established here. Spectron could be a good approach perhaps. https://www.electronjs.org/docs/tutorial/automated-testing-with-a-custom-driver  

But more then anything this repo would benefit from a rewrite in typescript, and an update to the core functions using electron APIs to support the latest version and features/options.

**Comment by furliq** [comment]
Agreed. Definitely more of a long term goal but at least you've laid out some of the first steps. Thanks again!
